published/external/xdpapi.h

```diff
@@ -311,6 +332,9 @@ XdpNmrClientAttachProvider(
             _Client->BindingHandle = _NmrBindingHandle;
             _Client->XdpApiProviderDispatch = (XDP_API_PROVIDER_DISPATCH *)_ProviderBindingDispatch;
             _Client->XdpApiProviderContext = _ProviderBindingContext;
+            KeSetEvent(&_Client->AttachEvent, 0, FALSE);
+        } else {
+            KePulseEvent(&_Client->AttachEvent, 0, FALSE);
```

Comment:
I don't think this needs to be a pulse, because we abandon the registration if anything fails.

Reply:
I just saw this from internal code. will remov